client/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,23 @@
+interface cardProps {
```

Comment:
This looks like we don't need it any more so can be deleted? I think if it can we should as it is could be confused with the PokemonCard

Reply:
Hi Jenny, Thanks for reviewing it. I assumed that I have closed PR related to Card Component in the path "client/src/components/Card/Card.tsx" as it's old one. The new PR created is the path "client/src/components/Card/PokemonCard.tsx". Sorry for the confusion here.

Reply:
So Card.tsx is not a valid component anymore.

Reply:
cool just delete and push to this repo - it will update this PR. You don't need to create another one.

Reply:
Thanks Jenny! Done now